# Returns the already optimized spline trajectory and footholds

---
OptimizedVariables x